fix allure reporting

programminx-askui opened the pull request.

Commit:
fix allure reporting

programminx-askui committed:
feat(): replace @askui/jest-allure-circus with allure-jest/node

programminx-askui committed:
Update README.md

mlikasam-askui commented on package.json:
```diff
@@ -39,21 +39,23 @@
     "@release-it-plugins/workspaces": "4.2.0",
     "@release-it/bumper": "6.0.1",
-    "@release-it/conventional-changelog": "9.0.2",
+    "@release-it/conventional-changelog": "^8.0.2",
```
was there a reason for the version change ?

adi-wan-askui replied:
+1

mlikasam-askui commented on package.json:
```diff
@@ -39,21 +39,23 @@
     "@types/jest": "29.5.3",
     "@types/jsdom": "21.1.1",
     "@types/node": "20.4.2",
+    "@types/ws": "^8.18.1",
```
was this added by you?

adi-wan-askui committed:
build: fix build

- remove syntax error in tsconfig-cjs.json
- upgrade release-it and plugins to be compatible with each other
  and new node versions

programminx-askui committed:
Merge pull request #31 from askui/fix-askui-report-fix-building

build: fix build
